### ISSUE TITLE
Whitelist rate limit on hub creation

### ISSUE THREAD

**Issue by camelgod** [mozilla/reticulum#614]
Many of us use Reticulum to create rooms programmatically from one endpoint.

The rate limit here 

https://github.com/mozilla/reticulum/blob/7fc356e3318f2f214acf67aec898d2c904fd7fad/lib/ret_web/controllers/api/v1/hub_controller.ex

makes it impossible to rapidly create rooms on high user demand..

Really need a IP whitelist or toggle to turn off and/or adjust the rate limiter.

**Comment by yakyouk** [comment]
You can work around this by checking the error code returned, 403 usually means you've been rate-limited and can retry

**Comment by camelgod** [comment]
> You can work around this by checking the error code returned, 403 usually means you've been rate-limited and can retry

Yeah but if you have more than one or lets say hundreds of requests per second? My server would be backed up. Its not exactly scaleable. Would be nice to be able to hammer the server in any case from the same IP.
